module LetError2
  fn test(): Int = {
    val x: Int = false;
    2
  }
end LetError2
